site/content/resources/entrepreneurship/shoplatinx.md

```diff
@@ -0,0 +1,9 @@
+---
+title: ShopLatinx
+image: "/assets/img/resources/shoplatinx.png"
```

Comment:
We need to make sure to add the image locally in the `src` directory

https://deploy-preview-299--techqueria.netlify.com/resources/entrepreneurship/

Reply:
But the way it is referenced here is good

Reply:
We're going to modifying the directory structure so this is easier to understand for future reference

Reply:
You got it, thanks!